playbooks/files/rax-maas/plugins/ceph_monitoring.py

```diff
@@ -39,10 +39,11 @@ def check_command(command, container_name=None):
     return json.loads(lines[-1])
 
 
-def get_ceph_rgw_hostcheck(rgw_host, rgw_port, container_name=None):
-    host_url = "http://%s:%s" % (rgw_host, rgw_port)
+def get_ceph_rgw_hostcheck(rgw_protocol, rgw_host, rgw_port,
+                           container_name=None):
+    host_url = "%s://%s:%s" % (rgw_protocol, rgw_host, rgw_port)
     try:
-        sc = requests.get(host_url).status_code
+        sc = requests.get(host_url, verify=False).status_code
```

Comment:
I'm personally not a fan of force cert acceptance, but I think we can follow on later to resolve this if we need.

Reply:
ditto - I heard that we use forced cert acceptance in some other calls, so I presumed there was a reason.

I'm making a follow up card in jira today to:

1. fix the rest of the hard-coded values (including hard-coded ports)
2. add a var where you can put cert/ca info if it's not a CA that can be verified by the system itself

Reply:
Me either... @supermari0 / @briancurtin can you make a future work follow on task for @dusty-simoni to review.

Reply:
👍

Reply:
https://rpc-openstack.atlassian.net/browse/TURTLES-313